src/main/java/com/prajina/academy/entity/Course.java

```diff
@@ -0,0 +1,84 @@
+package com.prajina.academy.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * @author prajina.purayil
+ *
+ */
+@Entity
+public class Course {
+
+	@Id
+	@GeneratedValue(strategy=GenerationType.SEQUENCE)
+	private Long id;
+	@Column(unique=true)
+	private String name;
+	private String desc;
```

Comment:
we usually define the @Column explicitly to avoid ambiguity, and enforce a standard naming conventions. 

@Column( name = "DESC")
private String description;